src/config.js

```diff
@@ -35,3 +45,8 @@ export const sidebar = {
     Parameters: 'parameters'
   }
 };
+
+export const runCommandTemplates = {
+  data: name => `kedro run --to-inputs ${name}`,
+  task: name => `kedro run --to-nodes ${name}`
+};
```

Comment:
I don't personally mind, but I thought you said that config.js shouldn't have variables in it? Should this be a utility? Though personally I'm happy for it to be in the meta sidebar component, because it's not very universal.

Reply:
Ah yeah so the thought is that configs shouldn't have things that:

  - have side effects
  - have conditionals
  - are non deterministic

So in other words they should be as static as possible, to avoid trouble.

Templates themselves are static, the only reason for the function here is because JS string templating needs it (I think), vs. e.g. `printf` style templates which don't.

Hopefully that makes sense as a compromise, but I could easily convert this to just use a string concat.

Reply:
ok sure thing. I'd still probably put it in the component (because it's not likely to be reused elsewhere in the app, and it's effectively just copy text), but no biggie

Reply:
Ah I've removed it from the config then.